### DISCUSSION TITLE
nuclei template

### DISCUSSION THREAD

**Discussion by abtinkr** [projectdiscovery/nuclei#4739] [General]
hey 

when I use nuclei like this " cat url.txt | nuclei -t /var/tmp/xss.yaml i have this error "[FTL] Could not run nuclei: no templates provided for scan" I tested many way and many templates but again this erorr, even though i tested default nucle-templates , I use 3.1.2 nuclei version and i removed and again installed but not changed, what can i do now?
i use Ubuntu server

**Comment by Mzack9999** [comment]
Could you execute again the command with verbose and debug flag and post the output:
```console 
cat url.txt | nuclei -t /var/tmp/xss.yaml -verbose -debug
```
Most likely there is an error in the template syntax

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
